sshd/issue_invite.sh

```diff
@@ -28,36 +28,37 @@ done
 shift $((OPTIND-1))
 
 TMP=`mktemp -d`
+npm install jsurl &>/dev/null
```

Comment:
Let's move this to the Dockerfile, no need to run it for each invocation of this script.

Hopefully we can do `npm install -g jsurl`.